apps/sugarfoot/src/workspaces/workspaces.controller.ts

```diff
@@ -52,21 +57,18 @@ export class WorkspacesController {
   }
 
   @Get(':id')
-  @UseGuards(WorkspaceMemberGuard)
   @ApiOperation({ summary: 'Get a workspace by ID' })
   @ApiResponse({ status: 200, description: 'Workspace details' })
   @ApiResponse({ status: 404, description: 'Workspace not found' })
   @ApiResponse({
     status: 403,
     description: 'Access denied - not a workspace member',
   })
-  findOne(@Param('id') id: string, @Request() req: AuthenticatedRequest) {
-    const userId = req.user.id;
-    return this.workspacesService.findOne(id, userId);
+  findOne(@Param('id') id: string) {
```

Comment:
As WorkspacePrivilegesGuard already checks if the user has privileges to the workspace, we don’t need the userId here anymore. I might remove more later